tests/admin_widgets/false_timezone_warning_templates/admin/base.html

```diff
@@ -0,0 +1,88 @@
+{% load i18n admin_static %}<!DOCTYPE html>
```

Comment:
Not sure -- maybe such a brittle test isn't worthwhile. We could add a comment in DateTimeShortcuts.js instead. I feel like this fix is probably only temporarily important anyway, i.e. going forward people overriding the template won't intentionally omit the attribute.